frontend/pages/SoftwarePage/SoftwareVulnerabilities/SoftwareVulnerabilitiesTable/SoftwareVulnerabilitiesTable.tests.tsx

```diff
@@ -250,8 +254,8 @@ describe("Software Vulnerabilities table", () => {
             has_next_results: false,
             has_previous_results: false,
           },
-          known_vulnerability: false,
         })}
+        emptyStateReason="unknown-cve"
         query="cve-2002-12345"
```

Comment:
does this need to be `'"unknown-cve"'` to get the double quotes into the string itself?

Reply:
Not that I know of, `emptyStateReason` is a string passed down and used as a key for `emptyStateDetails`